packages/api/src/resolvers/article/index.ts

```diff
@@ -399,13 +399,12 @@ export const getArticleResolver = authorized<
           'recommendations.recommender',
           'recommendations_recommender'
         )
-        .where('libraryItem.user_id = :uid', { uid })
 
       // We allow the backend to use the ID instead of a slug to fetch the article
       // query against id if slug is a uuid
       slug.match(/^[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}$/i)
-        ? qb.andWhere('libraryItem.id = :id', { id: slug })
-        : qb.andWhere('libraryItem.slug = :slug', { slug })
+        ? qb.where('libraryItem.id = :id', { id: slug })
```

Comment:
What does this change do?

Reply:
> What does this change do?

oh i see, we are removing the uid check above. Its a little scary to fully rely on RLS for that.

Reply:
The motive is to use the `slug` index to improve query performance. I think RLS should be quite reliable but alternatively we could update the index to use both `user_id` and `slug` columns too